data/language/nl-NL.yml

```diff
@@ -1446,7 +1446,7 @@ strings:
   1416: "{COLOUR WINDOW_2}Geproduceerd in vorige maand:"
   1417: "{COLOUR BLACK}{STRINGID} ({INT16}% vervoerd)"
   1418: "{COLOUR BLACK}{INT16}%"
-  1419: Closing down
+  1419: Sluit de deuren
```

Comment:
Ik kan deze vertaling niet zo rijmen. Foutje?

Reply:
Niet zo zeer, dit is dat een bedrijf de deuren sluit. Eventueel alternatief is "Aan het sluiten", maar ik wilde het duidelijker maken dat dit permanent is en "Permanent aan het sluiten" staat een beetje raar en "Failliet" dekt niet de originele lading.

Reply:
Ahh, dat verklaart 't. Oké, ik begrijp nu wat je bedoelt met 'Sluit de deuren', maar ik denk wel dat het te figuurlijk is. Misschien iets als 'Onder insolventie' of 'Onder curatele'? Denk dat insolventie passender is.

Reply:
Is dat niet te complex voor de doelgroep? Insolventie is nog ingewikkelder dan Faillissement.

Wellicht het gewoon versimpelen naar "Aan het stoppen".

Ik heb even gekeken wat nog andere alternatieven zijn:

“Einde in zicht”
“Deuren sluiten”
“Activiteiten gestaakt”
“Bedrijf opgeheven”
“Faillissement aangevraagd”
“Laatste dag”
“Gesloten voor goed”
“Einde verhaal”
“Lichten uit”
“Gordijnen dicht”

Ik vind "Activiteiten gestaakt" of "Bedrijf opgeheven" wel een mooie.

Reply:
> Is dat niet te complex voor de doelgroep? Insolventie is nog ingewikkelder dan Faillissement.

Haha, daar heb je wel gelijk in, ja.

> Ik vind "Activiteiten gestaakt" of "Bedrijf opgeheven" wel een mooie.

Die tweede vind ik ook wel een goede vondst. Laten we daarvoor gaan dan.

Reply:
Okidoki - wijziging aangebracht

Ik heb de changelog even laten zitten, ik zag geen precedent dat dat ook voor andere vertalingen werd gedaan.